sdk/core/core-http/src/util/xml.ts

```diff
@@ -62,9 +62,8 @@ xml2jsBuilderSettings.renderOpts = {
 
 /**
  * Converts given JSON object to XML string
- * @param obj JSON object to be converted into XML string
- * @param opts Options that govern the parsing of given JSON object
- * `rootName` indicates the name of the root element in the resulting XML
```

Comment:
This is a comment on a member of the options type, and this can be found in the comments on that member directly in the type definition of the option.